### DISCUSSION TITLE
X++ Development: Boost your Development Skills with Proven Tips and Tricks

### DISCUSSION THREAD

**Discussion by FH-Inway** [d365collaborative/d365fo.tools#778] [Show and tell]
Great session by @NathanClouseAX + @peter_ramer at the Community Summit 2023. Thanks for the d365fo.tools highlight and spreading the word, starting at https://youtu.be/XmiUfmTRNyo?t=1205. That's a great selection of cmdlets!

**Comment by NathanClouseAX** [comment]
Thanks! Full Slide Deck available at https://github.com/NathanClouseAX/CommunityPresentations/blob/master/2023/F%26O%20Summit%202023%20Speaker%20X%2B%2B%20Tips%20and%20Tricks%20Full.pptx
